### ISSUE TITLE
Auto pull recipes from conda forge

### ISSUE THREAD

**Issue by CJ-Wright** [NSLS-II/lightsource2-recipes#255]
It might be nice to have a system which trolls through conda forge for new versions of the recipes.

**Comment by danielballan** [comment]
Definitely! I don't like our current system of copy/paste, and this has been bothering me.

If we did adopt a tool for this it would have to be _really_ simple because we already have more complexity around the build system than we have the capacity to maintain.

**Comment by CJ-Wright** [comment]
Just for locality (this was in a PR comment), this is what I currently run:
```python
import os
from contextlib import contextmanager
@contextmanager
def indir(d):
    """Context manager for temporarily entering into a directory. Copyright (c) 2017, Anthony Scopatz
"""
    old_d = os.getcwd()
    ![cd @(d)]
    yield
    ![cd @(old_d)]
a = ['xpdsim', 'xpdtools', 'shed', 'xpdacq', 'xpdan']
t = 'https://raw.githubusercontent.com/conda-forge/{}-feedstock/master/recipe/meta.yaml'
for b in a:
    os.makedirs(b, exists_ok=True)
    with indir(b):
        if 'meta.yaml' in g`*`:
            rm meta.yaml
        wget @(t.format(b))
```
This plus Travis (running a nightly cron job) plus doctr to push the new recipes back to github could be all you needed to have a running system.

Only major changes here would be a revert of the deletion if the wget failed.

@justcalamari

**Comment by mrakitin** [comment]
May be worth to keep it open to see how we can use it for syncing between https://github.com/nsls-ii-forge and this repo before we fully abandon our internal builder.
